### DISCUSSION TITLE
Input field loses its focus when I use [q-route-tab] component

### DISCUSSION THREAD

**Discussion by saifobeidat** [quasarframework/quasar#8986] [General - Components / Directives / etc]
I have a page that contains an input field, I need it to be auto focused once you enter that page, so I am using this code:

**Search.vue page**
``` vue
<template>
  <input type="text" ref="focusMe"/>
</template>

<script>
export default {
  mounted(){
    this.$nextTick(() => {
      return this.$refs.focusMe.focus();
    })
  }
}

</script>
  
```

If you go to **Search.vue** page using the code, the auto focus will work:

``` vue
        <router-link
          :to="{
            name: 'search',
            query: {
              query: '',
              tab: 'lessons',
              country: selectedWebsiteCountryCode,
            },
            params: { lang: 'ar' },
          }"
          class="tab"
        >
          <i class="text-2xl fas fa-search"></i>
        </router-link>
```

![doesnt_lose](https://user-images.githubusercontent.com/24841143/115698234-12f8f380-a36d-11eb-8794-d3dcb9047f2e.gif)

But if you go to **Search.vue** page using the code, the auto focus will work for the first time only, and then it loses the focus:

``` vue
<q-route-tab
          :to="{
            name: 'search',
            query: {
              query: '',
              tab: 'lessons',
              country: selectedWebsiteCountryCode,
            },
            params: { lang: 'ar' },
          }"
          class="tab"
        >
          <i class="text-2xl fas fa-search"></i>
        </q-route-tab>
```

![focus_gets_losed](https://user-images.githubusercontent.com/24841143/115697777-8b12e980-a36c-11eb-9f34-70c79dc889eb.gif)

**Comment by ontwikkelfabriek** [comment]
Looks like the `mounted()` hook is not called the second time you click in your second example with `q-route-tab`. Is that correct?
